src/evict/evict_lru.c

```diff
@@ -594,12 +594,8 @@ __evict_pass(WT_SESSION_IMPL *session)
 		 * If there is a request to clear eviction walks, do that now,
 		 * before checking if the cache is full.
 		 */
-		if (F_ISSET(cache, WT_CACHE_CLEAR_WALKS)) {
-			F_CLR(cache, WT_CACHE_CLEAR_WALKS);
-			WT_RET(__evict_clear_walks(session));
-			WT_RET(__wt_cond_signal(
-			    session, cache->evict_waiter_cond));
-		}
+		if (F_ISSET(cache, WT_CACHE_CLEAR_WALKS))
```

Comment:
OK, for now let's rename it to something like `WT_CACHE_PASS_INTERRUPT` or `STOP` or similar.